presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -1,55 +1,63 @@
 ================
-Table statistics
+Table Statistics
 ================
 
-Presto supports statistics based optimizations for queries. For query to be subject for such optimization
-statistics information must be available for tables used by query.
+Presto supports statistics based optimizations for queries. For a query to take advantage of these optimizations,
+Presto must have statistical information for the tables in that query.
 
-Table statistics are provided to query planner by connectors.
-Currently the only connector supporting the mechanism is Hive connector.
+Table statistics are provided to the query planner by connectors.
+Currently the only connector that supports statistics is the Hive connector.
```

Comment:
Yeah - to some extent (only table level stats for now). I am not sure if that is important information for end users. As TPC-H connector is only used for testing purposes.